app-catalog/src/components/charts/EditorDialog.tsx

```diff
@@ -73,17 +89,22 @@ export function EditorDialog(props: {
   }
 
   useEffect(() => {
-    setChartInstallDescription(`${chart.name} deployment`);
-    fetchChartDetailFromArtifact(chart.name, chart.repository.name).then(response => {
-      if (response.available_versions) {
-        const availableVersions = response.available_versions.map(({ version }) => ({
-          title: version,
-          value: version,
-        }));
-        setVersions(availableVersions);
-        setSelectedVersion(availableVersions[0]);
-      }
-    });
+    if (globalThis.CHART_PROFILE === VANILLA_HELM_REPO) {
```

Comment:
Instead of using global variables for this, can you please pass a prop into EditorDialog?

Reply:
It looks like the global variables like CATALOG_NAME are still being used?